baseline.gradle

```diff
@@ -79,6 +79,7 @@ subprojects {
           '-Xep:StrictUnusedVariable:OFF',
           '-Xep:TypeParameterShadowing:OFF',
           '-Xep:TypeParameterUnusedInFormals:OFF',
+          '-Xep:ShutdownHook:OFF',
```

Comment:
Does this disable JVM Shutdown Hooks? I am a little confused that we are disabling this when we add a shutdown hook?

Reply:
It just avoid building failure for gradle.
As  https://github.com/palantir/gradle-baseline mentioned:
`ShutdownHook: Applications should not use Runtime#addShutdownHook.
`
and we can add this properties to skip this.